server/index.js

```diff
@@ -22,3 +23,42 @@ app.use(parser.json());
 app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 })
+
+// To render items in Bonus Tier (passed postman)
+app.get('/bonus/:bundleId', (req, res) => {
+  let bundleId = req.params.bundleId;
+
+  db.getBonuses(bundleId, (err, results) => {
+    if (err) {
+      res.send(err);
+    } else {
+      res.json(results);
+    }
+  })
+})
+
+// To render individual bonus upon click selection
+app.get('/bonus-title', (req, res) => {
+  let selectedTitle = req.query.title;
```

Comment:
Is there a reason to use query params in one case, here on line 42, and req params above, see line 29? If someone else has to hit these routes, then it may be confusing to have one served by one kind of parameter and the other with a different kind.

Reply:
Request params do not accommodate multi-word options (at least not the way I was testing it).

Reply:
camelCase?